src/components/Buzz/InteractiveTipBuzzButton.tsx

```diff
@@ -108,61 +109,47 @@ export function InteractiveTipBuzzButton({
   const theme = useMantineTheme();
   const currentUser = useCurrentUser();
   const features = useFeatureFlags();
+
   const [buzzCounter, setBuzzCounter] = useState(0);
   const timeoutRef = useRef<NodeJS.Timeout | null>(null);
+
   const interval = useInterval(() => {
     setBuzzCounter((prevCounter) => {
-      const [_, step] = steps.find(([min]) => prevCounter >= min) ?? [0, 10];
+      const [, step] = steps.find(([min]) => prevCounter >= min) ?? [0, 10];
       return Math.min(currentUser?.balance ?? 0, prevCounter + step);
     });
   }, 150);
-  const queryUtils = trpc.useContext();
+
   const onTip = useStore((state) => state.onTip);
   const [dismissed, setDismissed] = useLocalStorage({
     key: `interactive-tip-buzz-tutorial`,
     defaultValue: false,
   });
 
-  const createBuzzTransactionMutation = trpc.buzz.createTransaction.useMutation({
-    async onSuccess(_, { amount }) {
-      await queryUtils.buzz.getUserAccount.cancel();
-      if (entityType && entityId) {
-        onTip({ entityType, entityId, amount });
-      }
-
-      queryUtils.buzz.getUserAccount.setData(undefined, (old) =>
-        old
-          ? {
-              ...old,
-              balance: amount <= old.balance ? old.balance - amount : old.balance,
-            }
-          : old
-      );
-
-      if (onTipSent) {
-        onTipSent({ amount, queryUtils });
-      }
```

Comment:
I noticed that we never pass the `onTipSent`, so just ignored it @lrojas94 let me know if it's needed